velox/functions/sparksql/tests/StringTest.cpp

```diff
@@ -1055,5 +1055,58 @@ TEST_F(StringTest, empty2Null) {
   EXPECT_EQ(empty2Null(""), std::nullopt);
   EXPECT_EQ(empty2Null("abc"), "abc");
 }
+
+TEST_F(StringTest, varcharTypeWriteSideCheck) {
```

Comment:
Would you please add a separate file for the test as the StringTest.cpp is large?